src/feature/BoltFeature.ts

```diff
@@ -0,0 +1,66 @@
+import { ExtensionContext, commands, window, Uri } from 'vscode';
+import { IFeature } from '../feature';
+import * as path from 'path';
+import * as fs from 'fs';
+
+export class BoltConfigFeature implements IFeature {
+  dispose() {}
+  constructor(context: ExtensionContext) {
+    context.subscriptions.push(
+      commands.registerCommand('puppet-bolt.OpenUserConfigFile', () => {
+        let userInventoryFile = path.join(process.env['USERPROFILE'] || '~', '.puppetlabs', 'bolt', 'bolt.yml');
```

Comment:
Docs and code say the file is called `bolt.yaml`

https://github.com/puppetlabs/bolt/blob/b2a3511bf04f7cf9b42eeddff215b6ddcf75765b/lib/bolt/boltdir.rb#L34

https://puppet.com/docs/bolt/1.x/configuring_bolt.html

Reply:
👍